charts/nidhogg/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: nidhogg
 description: A Helm chart for Kubernetes
 type: application
-version: 0.1.0
-appVersion: "v0.5.0"
+version: v0.5.3
+appVersion: v0.5.3
```

Comment:
@chomatdam please confirm that we can remove those versions here now that the workflow should take care of it